airbyte-integrations/connectors/source-google-sheets/integration_tests/before.py

```diff
@@ -0,0 +1,77 @@
+"""
+MIT License
+
+Copyright (c) 2020 Airbyte
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+"""
+
+import json
+
+from apiclient import discovery
+
+from google_sheets_source.models.generated.spreadsheet import Spreadsheet, SpreadsheetProperties, Sheet, GridData, CellData, RowData, SheetProperties
+from google_sheets_source.helpers import Helpers
+
+# Override default permission scopes to allow creating and editing spreadsheets
+SCOPES = ['https://www.googleapis.com/auth/spreadsheets', 'https://www.googleapis.com/auth/drive', 'https://www.googleapis.com/auth/drive.file']
+SECRETS_PATH = "../secrets/config.json"  # TODO replace with injected secrets
+
+# TODO this whole file is one big TODO. placeholder code for pre and post actions when running integration tests
```

Comment:
can we get by without this and just host a single standard google sheet that all tests are run against?

Reply:
same for after.

Reply:
discussed offline. we prefer having before / after. sherif is going to take a look at how much work it is to add it to standard test source and make a judgmet call. we will add this feature to the standard test suite soon either way. just a question of whether it will happen before or after this pr.